backend/src/trainees/trainees.controller.ts

```diff
@@ -85,6 +110,49 @@ export class TraineesController {
     return assertNever(error);
   }
 
+  @ApiResponse({
+    type: TraineeAddNewResponseDto,
+  })
+  @ApiBody({ type: TraineeAddNewRequestDto })
+  @Post()
+  async create(
+    @Body() { trainee }: TraineeAddNewRequestDto,
+    @Request() req: ExpressRequest,
+  ): Promise<TraineeAddNewResponseDto> {
+    const isHttps = req.protocol === 'https';
+    const createTraineeCall = await this.traineesService.create(trainee);
+
+    if (!createTraineeCall.ok) {
+      const { error } = createTraineeCall;
+
+      if (error === 'TRAINEE_OR_USER_FOUND') {
+        throw new ConflictException(
+          'There is already a trainee which has the same pesel or an user with the same email',
```

Comment:
Quite an interesting grammar fact 😄 
https://grammarhow.com/a-user-or-an-user/

```suggestion
          'There is already a trainee which has the same pesel or a user with the same email',
```